package.json

```diff
@@ -21,13 +21,13 @@
     "@material-ui/core": "^3.1.0",
     "@material-ui/icons": "^3.0.1",
     "axios": "^0.18.0",
-    "bigwig-reader": "^1.0.9",
     "engine": "github:VALIS-software/Engine",
+    "fast-deep-equal": "^2.0.1",
+    "genomic-reader": "^1.4.7",
     "genomics-formats": "github:VALIS-software/Genomics-Formats",
+    "mkdirp": "^0.5.1",
```

Comment:
Was there a benefit to moving around `mkdirp` and `fast-deep-equal`? Just checking.

Also, it seems like the entire `package-lock.json` file shouldn't be deleted.

Reply:
I did not move it myself. npm did it.